frontends/web/src/components/amount/amount.test.tsx

```diff
@@ -19,79 +19,128 @@ import { render } from '@testing-library/react';
 import { Amount } from './amount';
 import { CoinUnit, ConversionUnit } from './../../api/account';
 
+const validateSpacing = (values: string[], elements: Element[]) => {
+  // each element in `values` is an expected
+  // "spaced" element. E.g:
+  // ["12"] means <span class="__space__random">12</span>.
+  
+  // We can't merely compare like so:
+  // expect(elements.innerHTML)toBe(<span class="space">12</span>, etc..)
+  // because the className gets "transformed".
```

Comment:
We used to just do this:
```js
expect(blocks.innerHTML).toBe(
          '<span class="">12</span>' +etc..
```

but as explained above and in the pr desc, vite seems to be generating a different css than CRA (classnames are randomised/hashed). Vite probably compiles it all to "actual HTML" then run the tests 🤷‍♂️ just a guess.